frontend/src/main/java/com/google/tripmeout/frontend/TripModel.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.tripmeout.frontend;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class TripModel {
+  public abstract String id();
+  public abstract String name();
+  public abstract long locationLat();
+  public abstract long locationLong();
+
+  public static Builder builder() {
```

Comment:
Nit: move the static builder method to the top of the class definition since it's the public entry point.